test/integration/Cover/expireCover.js

```diff
@@ -136,7 +136,7 @@ describe.skip('expireCover', function () {
     await submitIncident({ yc, productId, period, signer: this.accounts.defaultSender });
 
     // accept incident
-    await as.connect(staker1).castVotes([assessmentId], [true], parseEther('100'));
+    await as.connect(staker1).castVotes([assessmentId], [true], ['ipfsAssessmentDataHash'], parseEther('100'));
```

Comment:
I'd call it 'Assesment data hash' so it doesn't coincide exactly with the field name so there's no possibility of collision with a JS key.

Reply:
You can do a 1-shot search and replace with intellij if you have it replacing the string "ipfsAssessmentDataHash'" so you don't do it manually.

Hopefully that works.